CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - `set_user` action in sample constitutions correctly handles `user_data` (#4229).
 
+### Removed
+
+- Snapshots generated by 1.x services can no longer be used to join from or recover a new service (#4255).
```

Comment:
Let's add a note to make it clear that this means a 1.x ledger must effectively upgrade to 2.x first and produce at least one snapshot, before going to 3.x. Or that it is at the very least strongly recommended, unless the ledger is quite short.